ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/AttestationUtil.java

```diff
@@ -98,6 +98,113 @@
     return previous_epoch_attestations;
   }
 
+  public static List<PendingAttestation> get_previous_epoch_justified_attestations(
+      BeaconState state) {
+    // todo
+    return new ArrayList<PendingAttestation>();
+  }
+
+  public static List<Integer> get_previous_epoch_justified_attester_indices(BeaconState state) {
+    // todo
+    return new ArrayList<Integer>();
+  }
+
+  public static UnsignedLong get_previous_epoch_justified_attesting_balance(BeaconState state) {
+    // todo
+    return UnsignedLong.ZERO;
+  }
+
+  public static List<Integer> get_previous_epoch_boundary_attester_indices(BeaconState state) {
+    // todo
+    return new ArrayList<Integer>();
+  }
+
+  /**
+   * Returns the sum of balances for all the attesters that were active at the current epoch
+   * boundary
+   *
+   * @param state
+   * @return current_epoch_boundary_attesting_balance
+   */
+  public static UnsignedLong get_current_epoch_boundary_attesting_balance(BeaconState state)
+      throws Exception {
```

Comment:
Is there supposed to be a closed bracket here?

Reply:
I might not be following you here.  It wouldn't build if it was missing a brace